pixeltable/store.py

```diff
@@ -216,18 +216,30 @@ def create(self) -> None:
             # condition here?)
 
         # ensure that all visible indices exist by running Create Index If Not Exists
-        for index in self.sa_tbl.indexes:
-            create_stmt = sql.schema.CreateIndex(index, if_not_exists=True).compile(dialect=postgres_dialect)
-            self._exec_if_not_exists(str(create_stmt), wait_for_table=True)
+        if not Env.get().is_using_cockroachdb:
+            # TODO: This approach doesn't work on CockroachDB. (Do we need it to?)
```

Comment:
I don't understand this todo, let's discuss tomorrow.

Reply:
Ok. I don't understand your intent behind the new architecture for index creation (in particular how it's supposed to work with cockroach vector indices). Happy to discuss tomorrow